alica_ros_turtlesim/README.md

```diff
@@ -216,7 +216,7 @@ In this file, you need to implement the teleportation of the  turtles:
 follow the standard ros build step.
 ```
 cd catkin_ws
-source /opt/ros/<enter-your-ros-version-name-here>/setup.bash
+source /opt/ros/$ROS_DISTRO/setup.bash
```

Comment:
sorry yeah this wont be set if sourcing isnt done yet

Reply:
okay i fixed that ^^